From 9c29653dc6b0f4e9a74cd119d60a84ed8c4a6ec6 Mon Sep 17 00:00:00 2001
From: Stephen Smalley <stephen.smalley.work@gmail.com>
Date: Wed, 13 May 2026 09:59:08 -0400
Subject: [PATCH] sandbox/seunshare: fix getopt flags
Content-type: text/plain

-k, -v, and -C do NOT accept an argument, and the optstring
was not correct.

Signed-off-by: Stephen Smalley <stephen.smalley.work@gmail.com>
---
 sandbox/seunshare.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/sandbox/seunshare.c b/sandbox/seunshare.c
index a48c88036088..0dfc5cb2b820 100644
--- a/sandbox/seunshare.c
+++ b/sandbox/seunshare.c
@@ -776,10 +776,10 @@ int main(int argc, char **argv) {
 		{"homedir", 1, 0, 'h'},
 		{"tmpdir", 1, 0, 't'},
 		{"runuserdir", 1, 0, 'r'},
-		{"kill", 1, 0, 'k'},
-		{"verbose", 1, 0, 'v'},
+		{"kill", 0, 0, 'k'},
+		{"verbose", 0, 0, 'v'},
 		{"context", 1, 0, 'Z'},
-		{"capabilities", 1, 0, 'C'},
+		{"capabilities", 0, 0, 'C'},
 		{"wayland", 1, 0, 'W'},
 		{"pipewire", 1, 0, 'P'},
 		{NULL, 0, 0, 0}
@@ -811,7 +811,7 @@ int main(int argc, char **argv) {
 	}
 
 	while (1) {
-		clflag = getopt_long(argc, argv, "Ccvh:r:t:W:Z:", long_options, NULL);
+		clflag = getopt_long(argc, argv, "Ckvh:r:t:W:P:Z:", long_options, NULL);
 		if (clflag == -1)
 			break;
 
-- 
2.54.0

